AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -208,8 +208,18 @@ + (int)createTablesWithSchema:(MSDBSchema *)schema inOpenedDatabase:(void *)db {
 
   // Create the tables.
   if (tableQueries.count > 0) {
-    NSString *createTablesQuery = [tableQueries componentsJoinedByString:@"; "];
-    result = [self executeNonSelectionQuery:createTablesQuery inOpenedDatabase:db];
+    
+    /*
+     * Here, we do not join queries by ';', as before, because
+     * we do not execute a non-selection query using `exec`, as before.
+     * We are using `step`, and `step` can handle only one-line statements.
```

Comment:
rewrite this:

``` 
We do not join queries with ';' because we do not execute a non-selection query using `exec`. We are using `step`, and `step` can only handle one-line statements.
```